### PR TITLE
enemy_skill table additions

### PR DESCRIPTION
**Pull request by loopdeer**
- Created a new converter that outputs a new description for enemy skills. This is stored in the enemy_skills table in a column called "desc_en_emoji". "Emojis" are actually placeholder values such as {dispel} for end user replacement (whether that be images, html, etc.)
- data_exporter.py is edited to output the emoji text. (Note: this is more of a debug tool)
- enemy_skill.json is edited to correspond with the new column
- mysql.sql is edited to add the new column
- enemy_skill.py is edited for schema changes so it can read and store the desc_en_emoji with the new converter

### CODE REVIEW

File: etl/pad/raw/skills/emoji_en/enemy_skill_text.py [hunk 1]
```diff
@@ -0,0 +1,599 @@
+from pad.raw.skills.emoji_en.skill_common import *
+
+import logging
+
+human_fix_logger = logging.getLogger('human_fix')
```
Comment (RheingoldRiver) [inline etl/pad/raw/skills/emoji_en/enemy_skill_text.py:5]: Can the emoji additions happen in pad-cogs? either no emoji addition here at all or some kind of text replacement that we can replace in pad-cogs

Comment (loopdeer) [reply etl/pad/raw/skills/emoji_en/enemy_skill_text.py:5]: I can just use discord aliases like ":thumbsup:" or {thumbsup} depending on whichever one is safer

Comment (RheingoldRiver) [reply etl/pad/raw/skills/emoji_en/enemy_skill_text.py:5]: I left more comments in discord, but I don't think emojis period are appropriate here

**Comment by loopdeer** [comment]
Removed the enemy_skill_type column unless I can find a real reason to have it. Replaced the emojis with {placeholder}. No longer need UTF8MB4

File: etl/pad/raw/skills/emoji_en/active_skill_text.py [hunk 2]
```diff
@@ -0,0 +1,546 @@
+from pad.raw.skills.en.skill_common import *
```
Comment (chasehult) [inline etl/pad/raw/skills/emoji_en/active_skill_text.py:1]: You probably don't need this file or the leader skill one given that they're just copies of their text counterparts.

Comment (loopdeer) [reply etl/pad/raw/skills/emoji_en/active_skill_text.py]: done

**Comment by chasehult** [comment]
The following will need to be executed before merge:
```sql
ALTER TABLE enemy_skills ADD COLUMN desc_en_emoji text NOT NULL;
```

**Comment by chasehult** [comment]
> * Created a new converter that outputs the skill descriptions in the same condensed emoji format that is seen by Fagan bot.
> * Added a desc_en_emoji column and enemy_skill_type column (NOTE: These need to be added in whatever database Tsubaki is running, and I don't know if there is a file I modify here to add sql schema changes)
>   
>   * desc_en_emoji must use utf8mb4 in place of utf8 or else it will error out (I can also replace the emojis with discord aliases if this is not okay)
>   * enemy_skill_type is an INT
> 
> Final Note: Emojis are obviously not final, these are just the same ones I have been using as an example

Hey @loopdeer do you think you could rewrite the PR description to reflect all the changes you've made since you originally made the PR?

**Comment by loopdeer** [comment]
Rewrote the PR description

**Pull request merged**